### ISSUE TITLE
Coordinate conventions on wiki and tibiamaps

### ISSUE THREAD

**Issue by jatentaki** [tibiamaps/tibia-map-data#278]
TLDR: tibiamaps.io reports coordinates as 32321,32220,7 while tibiawiki as 126.65-125.221-7. I know how the tibiamaps format relates to downloadable file names but how does it work with tibiawiki?

Longer: I'm building a pathfinding tool. It works (locally, in python) and supports "teleports" autogenerated from the maps (stair squares are auto-linked with the one directly above/below whenever possible). This could be a very useful tool, for example to quickly generate quest walkthroughs. Unfortunately the current iteration lacks the myriad of fast transportation options, so I would like to make it coordinate-compatible with tibiawiki to make it easier to populate the database with ship captains, teleports etc.

**Comment by BourbonCrow** [comment]
`        function client_to_wiki(xy, mod) {
            return Math.floor(xy / 256) + "." + (xy % 256 + Number(mod));
        }

        function wiki_to_client(xy) {
            nums = xy.split(".");
            result = Number(nums[0]) * 256 + Number(nums[1]);
            return result;
        }`
this is the dode i found in their https://tibia.fandom.com/wiki/MapConvert page, i hope it helps
